src/locales/ko/dialogue.ts

```diff
@@ -854,22 +854,22 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "roark": {
     "encounter": {
-      1: "I need to see your potential as a Trainer. And, I'll need to see the toughness of the Pokémon that battle with you!",
-      2: "Here goes! These are my rocking Pokémon, my pride and joy!",
-      3: "Rock-type Pokémon are simply the best!",
-      4: "I need to see your potential as a Trainer. And, I'll need to see the toughness of the Pokémon that battle with you!"
+      1: "트레이너로서 너의 실력이 어느 정도인지 그리고 함께 싸울 포켓몬이 얼마나 강한지 확인해보겠어!",
+      2: "자 간다! 내 자랑이자 기쁨인 바위타입 포켓몬들이야!",
+      3: "바위타입 포켓몬이야말로 최고지!",
+      4: "트레이너로서 너의 실력이 어느 정도인지 그리고 함께 싸울 포켓몬이 얼마나 강한지 확인해보겠어!"
     },
     "victory": {
-      1: "W-what? That can't be! My buffed-up Pokémon!",
-      2: "…We lost control there. Next time I'd like to challenge you to a Fossil-digging race underground.",
-      3: "With skill like yours, it's natural for you to win.",
-      4: "Wh-what?! It can't be! Even that wasn't enough?",
-      5: "I blew it."
+      1: "이-이럴수가! 열심히 단련시킨 포켓몬들이!",
+      2: "…우리가 좀 지나쳤나 봐. 다음번엔 화석 발굴 경주를 해보고 싶군.",
```

Comment:
```suggestion
      2: "…져버린 건가. 다음엔 지하통로에서 화석캐기 승부를 하고 싶구나.",
```
약간 어색한 듯하여, [본가의 대사](https://youtu.be/jJgH5KkDkTg?si=Be9R4Bnrh9Sv8zWE&t=435)를 그대로 쓰는 걸 제안합니다. (4세대 대사와 5세대 대사가 거의 동일합니다.)

Reply:
감사합니다. 안그래도 이부분이 진짜 애매해서 정말 머리 아팠거든요. 5세대에 딱 괜찮은 대사가 있었네요!